extras/Linux-Cheatsheet-English.md

```diff
@@ -0,0 +1,24 @@
+# Linux Cheatsheet (English)
+
+| Description | Command |
+|---|---|
+| Clear terminal | `clear` |
+| Show path of the current directory | `pwd` |
+| Change into a directory | `cd <Path to directory>` |
+| List a directory | `ls -l <Path to directory>` |
+| Create directory | `mkdir <Path to directory>` |
+| Remove empty directory | `rmdir <Path to directory>` |
+| Remove directory | `rm -rf <Path to directory>` |
```

Comment:
Remove directory with force is maybe not the best option for unexperienced users.

`rm -ri` for interactive mode could be a solution.

Reply:
Agreed. Fixed it.